.github/workflows/ci.yaml

```diff
@@ -996,7 +996,10 @@ jobs:
             --durations=10 \
             -p no:sugar \
             --dburl=mysql://root:password@127.0.0.1/homeassistant-test \
-            tests/components/recorder
+            tests/components/history \
+            tests/components/logbook \
+            tests/components/recorder \
+            tests/components/sensor
```

Comment:
Just wondering: should these also be added to postgre or is that for a follow-up PR?

Reply:
That should indeed be added also to PostgreSQL too, I'll rebase and add that 👍